Assets/scss/components/article-cards.scss

```diff
@@ -0,0 +1,95 @@
+.article-cards{
+    background-color: $white-smoke;
+    font-family: $arial;
+    max-width: 100%;
+    margin: 0;
+
+    &__menu{
+        display: flex;
+        padding: 20px;
+       
+        &--item{
+            @include article-dropdown-style;
```

Comment:
I couldn't see where this article-dropdown-style is coming from. I'm possibly just missing something obvious

Reply:
@mathrik  article-dropdown-style is a mixin which is coming from the mixin.scss

![image](https://github.com/jvaishnavrp/FETraining/assets/107697604/1bc6ad32-8094-45fa-8bf5-0f25f06f9d64)

Reply:
Yeah, somehow I missed that mixins file. Thanks for pointing it out